src/EditorFeatures/Core/Implementation/Diagnostics/SuggestionTag.cs

```diff
@@ -1,32 +1,61 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
+using System.ComponentModel.Composition;
 using System.Windows.Media;
 using System.Windows.Shapes;
 using Microsoft.CodeAnalysis.Editor.Implementation.Adornments;
+using Microsoft.VisualStudio.Text.Classification;
 using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Utilities;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.Diagnostics
 {
+    [Export(typeof(EditorFormatDefinition))]
+    [Name(SuggestionTagFormat.ResourceName)]
+    [UserVisible(true)]
+    internal sealed class SuggestionTagFormat : EditorFormatDefinition
+    {
+        public const string ResourceName = "SuggestionTagFormat";
+
+        public SuggestionTagFormat()
+        {
+            this.ForegroundColor = Color.FromArgb(200, 0xA5, 0xA5, 0xA5);
+            this.BackgroundCustomizable = false;
+            this.DisplayName = EditorFeaturesResources.Suggestion_ellipses;
+        }
+    }
+
     /// <summary>
     /// Tag that specifies line separator.
     /// </summary>
     internal class SuggestionTag : GraphicsTag
     {
-        public static readonly SuggestionTag Instance = new SuggestionTag();
+        private Pen _graphicsTagPen;
 
-        private static Pen s_pen;
+        public SuggestionTag(IEditorFormatMap editorFormatMap) 
+            : base(editorFormatMap)
+        {
+        }
+
+        protected override Color? GetColor(
+            IWpfTextView view, IEditorFormatMap editorFormatMap)
+        {
+            var property = editorFormatMap.GetProperties(SuggestionTagFormat.ResourceName)["ForegroundColor"];
+            return property as Color?;
```

Comment:
Nope. But i don't trust VS.  And this is all sorts of dynamic dictionary lookups.  So i'm going to be resilient.